### PR TITLE
Updated comment for DefaultLoadBalancerName to provide further context

### PR DESCRIPTION
**Pull request by MorrisLaw**
**What this PR does / why we need it**:
Updates the comment for DefaultLoadBalancerName to provide better context and also as a reminder that it should eventually be removed.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cloudprovider/cloud.go [hunk 1]
```diff
@@ -62,8 +62,9 @@ type Clusters interface {
 	Master(ctx context.Context, clusterName string) (string, error)
 }
 
-// TODO(#6812): Use a shorter name that's less likely to be longer than cloud
-// providers' name length limits.
+// DefaultLoadBalancerName is for getting the load balancer name. It should eventually be removed
```
[the remark this answers is not in the document]
Comment (MorrisLaw) [reply pkg/cloudprovider/cloud.go]: I think this is good too

**Comment by andrewsykim** [comment]
/lgtm 
/approve

Thanks @MorrisLaw!

**Pull request merged**